test/built-ins/BigInt/asIntN/arithmetic.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN arithmetic test cases
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  3. Let mod be a BigInt representing bigint modulo 2**bits.
+  4. If mod ≥ 2**bits - 1, return mod - 2**bits; otherwise, return mod.
```

Comment:
I'm not sure if BigInt.asIntN is necessary to add here, maybe BigInt only. 

cc @rwaldron